ci-operator/step-registry/cucushift/installer/rehearse/aws/usgov/ipi/cucushift-installer-rehearse-aws-usgov-ipi-workflow.yaml

```diff
@@ -1,10 +1,10 @@
 workflow:
-  as: cucushift-installer-rehearse-aws-ipi-user-provisioned-dns
+  as: cucushift-installer-rehearse-aws-usgov-ipi
```

Comment:
If possible, suggest to keep `custom-dns` keyword in the workflow name, so that we can easily filter out the related install workflow.